### PR TITLE
Bugfixes -- camlaonam spawner / bases / etc rework

### PR DESCRIPTION
**Pull request by dijksterhuis**
ACAV Arsenal connected

ACAV heavy vehicles -- lowered

MPs cayuse -> slick

633rd moved the medical bird back slightly

633rd ch47 medical

ARVNR whitelisting for tanks

fix MP gate

mp slick

mp patrol vehicle

blackhorse 105s at base not behsdfkjhdsfsdfstanden

Blackhorse -- Removed Berch assets

blackhorse -- slicks too

mp_arsenal no longer safe zone

spread out acav vehicles

anzac new billboard

arvnr -- arvn specific spawners to fix whitelisting issues

Muskets -- OH6A spawn fixed

7thCav -- OH6A spawn fixed

teleporters -- fixed MP missing MACV and removed berch from list of options to TP to

### CODE REVIEW

**Comment by dijksterhuis** [comment]
testing

- ✅ ACAV Arsenal connected
- ✅ ACAV heavy vehicles -- lowered
- ✅ MPs cayuse -> slick
- ✅ 633rd moved the medical bird back slightly
- ✅ 633rd ch47 medical
- ✅ ARVNR whitelisting for tanks
- ✅ fix MP gate
- ✅ mp slick
- ✅ mp patrol vehicle
- ✅ blackhorse 105s at base not behsdfkjhdsfsdfstanden
- ✅ Blackhorse -- Removed Berch assets
- ✅ blackhorse -- slicks too
  - they've also got access to choktaws / seahorses now?
- ❌ mp_arsenal no longer safe zone
  - i'm gonna dig into this, should be working
- ✅ spread out acav vehicles
- ✅ anzac new billboard (after PR into your PR anyway)
- ✅ arvnr -- arvn specific spawners to fix whitelisting issues (wasn't the spawn issue iirc)
- ✅ Muskets -- OH6A spawn fixed
- ✅ 7thCav -- OH6A spawn fixed
- teleporters 
  - ✅ fixed MP missing MACV
  - ✅ removed berch from list of options to TP to
  
----

  todo (i'll PR onto this PR or you can fix if i've not done it -- read: fallen asleep)
  
  - there's an extra `}` character in the teleports config on like line 29
  - mp arsenal
  - comms (MP Base changes right after we've been saying everyone needs to wait 😁 )
  - anzac billboard paa

**Comment by dijksterhuis** [comment]
testing

✅ ACAV Arsenal connected
✅ ACAV heavy vehicles -- lowered
✅ MPs cayuse -> slick
✅ 633rd moved the medical bird back slightly
✅ 633rd ch47 medical
✅ ARVNR whitelisting for tanks
✅ fix MP gate
✅ mp slick
✅ mp patrol vehicle
✅ blackhorse 105s at base not behsdfkjhdsfsdfstanden
✅ Blackhorse -- Removed Berch assets
✅ blackhorse -- slicks too
✅  mp_arsenal no longer safe zone
✅ spread out acav vehicles
✅ anzac new billboard (after PR into your PR anyway)
✅ arvnr -- arvn specific spawners to fix whitelisting issues (wasn't the spawn issue iirc)
✅ Muskets -- OH6A spawn fixed
✅ 7thCav -- OH6A spawn fixed

teleporters
✅ fixed MP missing MACV
✅ removed berch from list of options to TP to

File: maps/cam_lao_nam/mission.sqm [hunk 1]
```diff
@@ -211,104 +212,111 @@ class AddonsMetaData
 			url="https://www.arma3.com";
 		};
 		class Item19
+		{
```
Comment (cerebral-314) [inline maps/cam_lao_nam/mission.sqm]: Please unload ALL mods before working on the map.
Remove this dependency please.

**Review by cerebral-314 — CHANGES_REQUESTED** [review]
Aside from the CBA dependency everything looks good.

**Review by cerebral-314 — APPROVED** [review]

**Comment by cerebral-314** [comment]
LGTM

**Comment by dijksterhuis** [comment]
@cerebral-314 this was an earlier version of the one @wescottj98 merged prior so gonna close this.

(I messed up the push so had to create a new PR)

**Pull request closed without merging**